From: =?utf-8?q?Jonas_Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Tue, 8 Mar 2022 12:55:00 +0100
Subject: magnifier: Update zoom region right after enabling the magnifier

We want the zoom region to always be where the mouse cursor is, so make
sure to update it initially, too.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2234>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/7bf8f40c
---
 js/ui/magnifier.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/js/ui/magnifier.js b/js/ui/magnifier.js
index 0585e88..8ff062b 100644
--- a/js/ui/magnifier.js
+++ b/js/ui/magnifier.js
@@ -215,6 +215,8 @@ var Magnifier = class Magnifier {
         if (!this._pointerWatch) {
             let interval = 1000 / 60;
             this._pointerWatch = PointerWatcher.getPointerWatcher().addWatch(interval, this.scrollToMousePos.bind(this));
+
+            this.scrollToMousePos();
         }
     }
 
